Library/UseCases/SearchFiltersUseCase.swift

```diff
@@ -224,15 +228,33 @@ private func filterPills(
       SearchFilterPill(
         isHighlighted: state != SearchFiltersUseCase.defaultStateOption,
         filterType: .projectState,
-        // FIXME: MBL-2218 Turn the state into a user-readable title.
-        buttonType: .dropdown(state.rawValue)
+        // FIXME: MBL-2232 Final translations
+        buttonType: .dropdown(hasState ? state.title : "Project status")
```

Comment:
FYI it looks like this string will be pulled in with my [Tracking Number Translations PR](https://github.com/kickstarter/ios-oss/pull/2389)